README.md

```diff
@@ -6,21 +6,10 @@
 [![Dependency Status](https://img.shields.io/david/cgiffard/node-simplecrawler.svg)](https://david-dm.org/cgiffard/node-simplecrawler)
 [![devDependency Status](https://img.shields.io/david/dev/cgiffard/node-simplecrawler.svg)](https://david-dm.org/cgiffard/node-simplecrawler#info=devDependencies)
 
-Simplecrawler is designed to provide the most basic possible API for crawling
-websites, while being as flexible and robust as possible. I wrote simplecrawler
-to archive, analyse, and search some very large websites. It has happily chewed
-through 50,000 pages and written tens of gigabytes to disk without issue.
-
-#### Example (simple mode)
```

Comment:
I reckon throwing some kind of basic code example at the very top might be useful, just to help people evaluate whether this is the right package for them, at a glance.